lib/broken_record/scanner.rb

```diff
@@ -36,7 +36,7 @@ def classes_to_validate(class_names)
     end
 
     def load_all_active_record_classes
-      Dir.glob(Rails.root.to_s + '/app/models/**/*.rb').each { |file| require file }
+      Dir.glob(Rails.root.to_s + '/app/models/**/*.rb').each { |file| require_dependency file }
```

Comment:
Actually, check out https://github.com/nickgervasi/broken_record/pull/9. This eliminates the `require`/`require_dependency` call altogether.